docs/data-collection/participant-prep.md

```diff
@@ -225,6 +232,9 @@
 
     ![send_isocenter](../assets/images/send_isocenter.png)
 
+- [ ] Once the participant is lying on the scanner bed, check that no arms/legs rest on the GA or the RB tubes and may block them.
+- [ ] Before continuing with the setup, make sure all cables and tubes leave the scanner's bed *perpendicularly* and lie on the floor.
```

Comment:
@celprov @esavary I kind or recall this is important? @smoia mentioned that the tubes should go on the floor, but I believe the ECG cable is better perpendicular to the bed to reduce artifacts, is that correct?
```suggestion
- [ ] Before continuing with the setup, make sure all cables and tubes leave the scanner's bed *perpendicularly* and lie on the floor.
    Tape them to the floor so that they don't move accidentally.
```

Reply:
Also - we should probably stick them to the floor with tape right? (added in my last commit)

Reply:
@oesteban I only remember the importance of the tube laying on the floor, and I don't remember the mention of taping the cables to the floor, but I never participated in a session with Stefano, so it is likely inaccurate.

Reply:
I'm referring to ensuring the ECG cable goes out of the room perpendicularly to the bed. Taping them is just something I believe would help make sure nothing moves

Reply:
Let's discuss this in #117.